### ISSUE TITLE
Audit loading times of all elements of disasterrisk.af

### ISSUE THREAD

**Issue by gmolinario** [GFDRR/geonode-afghanistan#11]
There is potentially a need to create offline syncing geonodes as well as a "light" online interface. 

We need to have an audit/report to understand what are the current loading times and bandwidth bottlenecks of the current geonode. Once the limitations are understood, we are requesting geosolutions to provide us a proposal on how to create a lighter loading interface, and for example improve the transactional speed on get map calls & caching .

**Comment by simboss** [comment]
An initial audit has been done and improvements have been performed on the new DisasterRisk instance.
[This](https://github.com/GFDRR/geonode-afghanistan/files/3637062/GTmetrix-report-disasterrisk.af-20190920T002231-GIv5eR0o-full.pdf) is an analysis on the current instance and [this](https://github.com/GFDRR/geonode-afghanistan/files/3637066/GTmetrix-report-disasterrisk-af-dev.geo-solutions.it-20190920T002329-YnNsclZa-full.pdf) is the analysis on the update instance where a good set of optimizations have been already performed.

**Comment by simboss** [comment]
Ok, we did some more tuning to do basic improvements, let us know if this helps with respect to the old instance.

We will try to do some more in the upcoming week.

**Comment by simboss** [comment]
Adding more info to what we have done together with some recommendations for the future.

Number 1 issue we found was the size of the images used for the template. The overall weight was around 10MB, so we reduced and optimized them.

Next we checked that GZIP compression was enabled everywhere for text, html, css and JS.

We also re-minified all the JS and made sure to remove a few unnecesary ones.

We will do another round shortly to go beyond these simple steps.
